pkg/preflight/validate.go

```diff
@@ -98,6 +109,34 @@ func validateVolumeAccessModes(ctx context.Context, l *log.Logger, client k8scli
 	return volAccessModeFailures, nil
 }
 
+// validateStorageClasses returns any failures encountered when discovering the source and destination
+// storage classes
+func validateStorageClasses(ctx context.Context, l *log.Logger, clientset k8sclient.Interface, sourceSCName, destSCName string) ([]ValidationFailure, error) {
```

Comment:
Could you copy the logic from the migrate version more precisely? This drops support for the 'skip-source-validation' flag

Reply:
done